src/status_im2/contexts/communities/menus/generic_menu/view.cljs

```diff
@@ -0,0 +1,21 @@
+(ns status-im2.contexts.communities.menus.generic-menu.view
+  (:require [utils.i18n :as i18n]
+            [quo2.core :as quo]
+            [status-im2.contexts.communities.menus.generic-menu.style :as style]
+            [react-native.core :as rn]
+            [utils.re-frame :as rf]))
+
+(defn view
```

Comment:
same menu is used for view rules, leave community and probably many other communities related bottom sheets. So I refactored to a generic component with title, context tag and children